doc/release/1.15.0-notes.rst

```diff
@@ -34,6 +34,15 @@ New Features
 Improvements
 ============
 
+``histogram`` and ``histogramdd` functions have moved to ``np.lib.histograms``
+------------------------------------------------------------------------------
+These were originally found in ``np.lib.function_base``. They are still
+available under their un-scoped ``np.histogram(dd)`` names, and
+to maintain compatibility, aliased at ``np.lib.function_base.histogram(dd)``.
+
+Code that does ``from np.lib.function_base import *`` will need to be updated
+with the new location, and should consider not using ``import *`` in future.
```

Comment:
Presumably we can get away with this? The alternative is to have duplicate entries in `__all__`, but we already do that...

Reply:
Agreed with this.